examples/wasm/wasm_sample/Makefile

```diff
@@ -0,0 +1,98 @@
+#sometimes there might not be a wasm-ld (Ubuntu:focal)
+#lets check if we can find a specific version
+
+ifneq ($(shell (command -v wasm-ld)),)
+  WASM-LD ?= wasm-ld
+  else
+    ifneq ($(shell (command -v wasm-ld-11)),)
+    LLVM_VERSION = 11
+    else
+      ifneq ($(shell (command -v wasm-ld-10)),)
+      LLVM_VERSION = 10
+      else
+        ifneq ($(shell (command -v wasm-ld-9)),)
+        LLVM_VERSION = 9
+        else
+          ifneq ($(shell (command -v wasm-ld-8)),)
+          LLVM_VERSION = 8
+          endif
+      endif
+    endif
+  endif
+endif
+
+ifneq ($(LLVM_VERSION),)
+  CLANG ?= clang-$(LLVM_VERSION)
+  CLANGPP ?= clang++-$(LLVM_VERSION)
+  WASM-LD ?= wasm-ld-$(LLVM_VERSION)
+else
+  CLANG ?= clang
+  CLANGPP ?= clang++
+endif
+
+WASM-LD ?= echo !! NO wasm-ld(-VERSION) found !!; false
```

Comment:
```suggestion
WASM-LD ?= echo "!! NO wasm-ld(-VERSION) found !!"; false
```

?

Reply:
print an error message when trying to link but no wasm-ld was found